src/routes/routes.js

```diff
@@ -0,0 +1,91 @@
+import { Router } from 'express';
+import Joi from 'joi';
+import validator from 'express-joi-validation';
+
+import UsersController from '../controllers/controller';
+
+const controller = new UsersController();
+const router = Router();
+const userValidator = validator.createValidator({ passError: true });
+
+const querySchema = Joi.object({
+  login: Joi.string().required(),
+  password: Joi.string().required(),
+  age: Joi.number()
+    .min(4)
+    .max(130)
+    .required(),
+});
+
+router
+  .get(
+    '/users/:id',
+    (req, res) => {
+      const user = controller.getUser(req.params.id);
+
+      if (user) {
+        res.json({ data: user });
+        return;
+      }
+
+      res.status(404).json({ message: 'A user with this ID was not found.' });
+    },
+  )
+  .put(
```

Comment:
User should be able to update login only, or age only, now all 3 fields are required